src/ch05/15.py

```diff
@@ -0,0 +1,36 @@
+import pylab
+import numpy as np
+from random import *

```

Comment:
Prevent using `from ... import *`.

```suggestion
from random import random
```